README.md

```diff
@@ -68,18 +67,31 @@ Note: This is an example of a non-normative note.
 
 # Overview
 
-Serving GraphQL over HTTP provides the ability to use the full advantages of GraphQL with the rich feature set of HTTP. Carrying GraphQL in HTTP does not mean that GraphQL overrides existing
-semantics of HTTP but rather that the semantics of GraphQL over HTTP map naturally to HTTP semantics.
+Serving GraphQL over HTTP provides the ability to use the full advantages of GraphQL with the rich feature set of HTTP.
+Carrying GraphQL in HTTP does not mean that GraphQL overrides existing semantics of HTTP,
+but rather that the semantics of GraphQL over HTTP map naturally to HTTP semantics.
 
-GraphQL naturally follows the HTTP request/response message model providing a GraphQL request in an HTTP request and
-GraphQL response in an HTTP response.
+GraphQL naturally follows the HTTP request/response message model,
+providing a GraphQL request in an HTTP request and GraphQL response in an HTTP response.
 
 # URL
 
-A GraphQL server operates on a single URL and all GraphQL requests for a given service should be directed
-at this URL. Other protocols may also use that URL.
+A GraphQL over HTTP compliant server MUST designate at least one URL that handles GraphQL requests.
+
+All GraphQL operations that are available in a schema MUST be available on all URLs that are designated to it.
+That means the client can send all their GraphQL requests to a single endpoint, regardless
+of the contained Operations, Fragments or Variables.
+
+Multiple URLs MAY exist on a server to handle GraphQL requests, potentially serving different
+but independent schemas. The same GraphQL schema MAY be available on multiple URLs on the server.
```

Comment:
```suggestion
but standalone schemas. The same GraphQL schema MAY be available on multiple URLs on the server.
```

Other ideas:
- Self-sufficient
- Complete

Maybe we don't need this section at all, but would rather specify cleanly what a single URL does.